packages/frontend/src/features/metricsCatalog/components/MetricsCatalogColumnName.tsx

```diff
@@ -150,28 +177,31 @@ export const MetricsCatalogColumnName: FC<Props> = ({ row, table }) => {
                 <ActionIcon
                     ref={setIconRef}
                     variant="default"
-                    w={25}
-                    h={25}
-                    radius="md"
-                    p="two"
-                    sx={{ flexShrink: 0 }}
                     onClick={handleIconClick}
+                    sx={(theme) => ({
+                        width: 28,
+                        height: 28,
+                        flexShrink: 0,
+                        borderRadius: '8px',
+                        border: `1px solid ${theme.colors.gray[3]}`,
+                        ...(!isEmojiIcon(row.original.icon) && {
+                            boxShadow:
+                                '0px -2px 0px 0px rgba(10, 13, 18, 0.07) inset, 0px 1px 2px 0px rgba(16, 24, 40, 0.05)',
+                        }),
+                    })}
                 >
                     {isEmojiIcon(row.original.icon) ? (
-                        <Emoji size={22} unified={row.original.icon.unicode} />
+                        <Emoji size={18} unified={row.original.icon.unicode} />
                     ) : (
-                        <MetricIconPlaceholder
-                            strokeWidth={1.5}
-                            width={22}
-                            height={22}
-                        />
+                        <MetricIconPlaceholder width={12} height={12} />
```

Comment:
styles applied as requested by @PriPatel 

and decreased icon placeholder's size